qf-faucet-server/src/main.rs

```diff
@@ -0,0 +1,288 @@
+use std::str::FromStr;
+
+use subxt;
+use subxt::{OnlineClient, PolkadotConfig};
+use subxt_signer::bip39::Mnemonic;
+use warp::{reject::Reject, Filter};
+use subxt_signer::sr25519::dev;
+use subxt_signer::sr25519;
+use std::env;
+use hex;
+use clap::Parser;
+use subxt::utils::{MultiAddress, AccountId32};
+use rusqlite::{Connection, Result as SqlResult};
+use chrono::Utc;
+
+
+#[subxt::subxt(runtime_metadata_path = "metadata.scale", derive_for_all_types = "PartialEq, Eq")]
+pub mod polkadot {}
+
+const RPC_URL: &str = "wss://dev.qfnetwork.xyz/socket";
+const FAUCET_AMOUNT: u128 = 20_000_000_000;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+struct DripRequest {
+    address: String,
```

Comment:
It's unsafe to accept unvalidated string from users. Especially if we put that in an SQL statement. Let's deserialize/validate it here into account address:
```suggestion
    address: AccountId32,
```

Reply:
from client side we get [u8; 64] as hex and after decode it to [u8; 32] AccountId32